examples/differentiable_push.py

```diff
@@ -86,15 +86,20 @@ def main():
     )
 
     ########################## build ##########################
-    scene.build()
+    scene.build(n_envs=2)
 
     ########################## forward + backward twice ##########################
+    v_list = []
+    w_list = []
+    horizon = 150
+    for i in range(horizon):
+        v_i = gs.tensor([[0.0, 1.0, 0.0], [0.0, 1.0, 0.0]], requires_grad=True)
```

Comment:
just use list comprehension:
`v_list  = [gs.tensor([[0.0, 1.0, 0.0], [0.0, 1.0, 0.0]], requires_grad=True) for _ in range(horizon)]`